docs/samples/transformer/image_transformer/README.md

```diff
@@ -9,17 +9,19 @@ Most of the model servers expect tensors as input data, so a pre-processing step
 
 ##  Build Transformer image
 
-### Extend Transformer and implement pre/post processing functions
+### Extend KFModel and implement pre/post processing functions
```

Comment:
We are still extending Transformer

Reply:
En, Seems the transformer is deleted now.

Reply:
Yes -- there is only one base class now. The default behavior of KFModel is to do forwarding if predictor/explainer host is provided. This is because the Transformer/Explainer abstract classes were effectively the same.

Reply:
This is my understanding too, thanks for confirm.